docs/config-file/v2.rst

```diff
@@ -512,6 +517,43 @@ Note that 0 means *normal rank*, not *no rank*.
    Is better to decrease the rank of pages you want to deprecate,
    rather than increasing the rank of the other pages.
 
+search.ignore
+`````````````
+
+Don't index files matching a pattern.
+This is, you won't see search results from these files.
+
+:Type: ``list`` of patterns
+:Default: ``['search.html', 'search/index.html', '404.html', '404/index.html']``
+
+Patterns are matched against the final html pages produced by the build
+(you should try to match `index.html`, not `docs/index.rst`).
```

Comment:
How is the pattern to use if the project is built with pretty URLs? `/` for index?  `/path/to/page/` for another page? This may be explained in the docs as well.

Reply:
From the defaults, I would guess that if I want to ignore `/404/`, I will need to put `/404/index.html`. If that's correct, we definitely need to explain this because it will bring lot of confusions.

Reply:
Also, having the URL disassociated from the path you need to put here is not good UX, IMO. However, I'm not sure how we can improve that. Is is possible to just use URLs here instead of path files?

Reply:
We index content from paths, so makes sense to use paths, as we do with rankings.

Reply:
> We index content from paths, so makes sense to use paths

I agree that makes sense from our side. I'm thinking if it's the same from the user's perspective. They see a URL like `/en/latest/404/` and they need to know that `404/index.html` is the correct value, which looks completely different than the URL.

Reply:
I don't thing is worth it to try to guess the correct path, that introduces a lot of problems rather than helping, we already ask the users to match the final path rather than the source file.